### ISSUE TITLE
[FEATURE] Animated Doors

### ISSUE THREAD

**Issue by xCraimax** [SBDPlugins/V10Lift#86]
**Is your feature request related to a problem? Please describe.**
/

**Describe the solution you'd like**
Would be cool, if the doors would be animated, when the realistic-mode is enabled.
I've thought, that this was already implemented, but seems not like it.

**Describe alternatives you've considered**

**Additional context**
![image](https://github.com/SBDPlugins/V10Lift/assets/49165798/d17b3b53-357b-4a25-ad36-7dbf6774ad3c)

**Comment by stijnb1234** [comment]
What do you think of an integration with https://www.spigotmc.org/resources/big-doors.58669/ instead?

**Comment by xCraimax** [comment]
Good idea, but big doors use more space & aren't as easy to time, in my opinion.

**Comment by stijnb1234** [comment]
Hmm okay, so you would like to see a door that's opening in steps, so no 'real' armorstand animations but still a bit of movement.
